docs/2.7.0/docs/daml-finance/tutorials/getting-started/lifecycling.rst

```diff
@@ -144,6 +144,22 @@ process up to the implementation.
 * The party executing settlement can be chosen as well, as described in the previous tutorial on
   :doc:`Settlement <settlement>`.
 
+Which party should take the role as *lifecycler*?
+=================================================
+
+From a design perspective, a lifecycler should be the party that defines the lifecycle events
```

Comment:
The event can be defined by parties other than the life-cycler.

The life-cycler is currently trusted with
- timely and complete event processing
- providing accurate observations

Reply:
I added both points to the answer now.